tests/nested/manual/muinstaller-core/task.yaml

```diff
@@ -217,11 +215,12 @@ execute: |
   {
     "snaps": ["snap-with-comps"],
     "components": {
-      "snap-with-comps": ["comp1"];
+      "snap-with-comps": ["comp1"]
     }
   }
   EOF
-  diff -u <(jq --sort-keys . available.json) <(jq --sort-keys . expected-available.json)
+  # gojq sorts keys by default
+  diff -u <(gojq  . available.json) <(gojq . expected-available.json)
```

Comment:
this is failing now, https://github.com/canonical/snapd/actions/runs/11932568734/job/33258400731#step:17:4097 looks like this may have never worked? @andrewphelpsj can you take a look?

Reply:
Hrm yeah must have been something with that semicolon maybe ruining things. I'll push a fix once I have it.

Reply:
thanks